lib/views/pages/join_organization.dart

```diff
@@ -51,34 +40,14 @@ class _JoinOrganizationState extends State<JoinOrganization> {
     } else if (!result.hasException && !result.loading) {
        _successToast("Sucess!");
       print(result.data);
-      Future.delayed(const Duration(milliseconds: 3000), () {
+      Future.delayed(const Duration(milliseconds: 2000), () {
```

Comment:
Why is this delay here?

Reply:
artificial delay is simply so the user will be able to see the success toast on that same page before going forward. Lmk what you think...

To filter by public or private organizations? There is a bool value that was added to organization so I can check if its public or not. I'll add that in later.